tools/build_variables.py

```diff
@@ -233,6 +233,8 @@ def add_torch_libs():
         "torch/csrc/onnx/init.cpp",
         "torch/csrc/serialization.cpp",
         "torch/csrc/tensor/python_tensor.cpp",
+        "torch/csrc/perf_tools/init.cpp",
+        "torch/csrc/perf_tools/ThroughputBenchmark.cpp",
```

Comment:
throughput_benchmark.cpp (sorry for the nit, but all other files in csrc seem to be consistent that way)